### PR TITLE
Make deployment to public cloud a bonus

### PR DESCRIPTION
**Pull request by germangh**
Reasons for this change [here](https://findhotel.slack.com/archives/G01H8F03A8K/p1658815201578759).

### CODE REVIEW

**Review by sergioaugrod — APPROVED** [review]

**Review by nvieirafelipe — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,15 +36,14 @@ In doing so:
...
 - A library/component that packages the import service and the interface for accessing the geolocation data;
 - A REST API application that uses the aforementioned library
-- Deploy the project on a cloud platform of your choice (e.g. [Gigalixir](https://www.gigalixir.com/), [Heroku](https://www.heroku.com/), [Fly.io](https://fly.io/), [Digital Ocean](https://www.digitalocean.com/), AWS, Google Cloud or Azure).
-- Import part of the sample data in the deployed application (according to the could platform limits).
 
 #### Bonus
 
 These items can add extra points to your evaluation, but you are not expected to work on them.
 
 - The API application is Dockerised.
```
Comment (prodis) [inline README.md:44]: Should this be a requirement and not a bonus?

File: README.md [hunk 2]
```diff
@@ -36,15 +36,14 @@ In doing so:
 
 - A library/component that packages the import service and the interface for accessing the geolocation data;
 - A REST API application that uses the aforementioned library
```
Comment (prodis) [inline README.md:38]:
```suggestion
- A REST API application that uses the aforementioned library.
```

File: README.md (hunk 1 above)
Comment (germangh) [reply README.md:44]: Yes I think so, but it's best you decide. You can go ahead and change it on this branch.

**Comment by germangh** [comment]
@prodis I made the dockerisation of the application mandatory. If we leave deployment as a bonus I think it makes sense to make the dockerisation mandatory and in general it should take for most people less effort to dockerise the app than to deploy it in a public cloud.

**Review by zekus — APPROVED** [review]

**Pull request merged**